src/index.ts

```diff
@@ -24,24 +25,25 @@ interface PersonalDataHeaders {
  */
 export function createPersonalDataHeaders(
   url: string,
-  req: Request | APIGatewayProxyEvent,
+  req: Request | APIGatewayProxyEvent
 ): PersonalDataHeaders {
   const domain = new URL(url).hostname;
   const personalDataHeaders: PersonalDataHeaders = {};
 
-  const txmaAuditEncodedHeader = req.headers[HEADERS.HEADER_TXMA];
+  const txmaAuditEncodedHeader = getHeader(req, HEADERS.HEADER_TXMA) as string;
```

Comment:
Out of interest: 
Why are we saying `as string`? The function returns `string | undefined` which is suitable for the rest of the function.

Reply:
Ah good question, made me realise I'd forced the return type from `getHeader` to be `any` which was not very helpful!

About to push a commit that takes that forced return type away. Without that it should have the correct return type based on `req.headers`. The reason to do `as string` here is because for some reason the type of the the headers values is `string | string[] | undefined`.

Arguably I should just be handling the possibility of it being `string[]` to be honest, but given we have generated the header and we know what it has in it it seems ok to Assert it as a `string`.